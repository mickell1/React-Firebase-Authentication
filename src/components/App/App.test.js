import React from 'react';
import { shallow } from 'enzyme';
import App from '../App';

describe('AreaChart', () => {
  it('should render AreaChart component', () => {
    const wrapper = shallow(<App />);
  
    expect(wrapper).toMatchSnapshot();
  });
});