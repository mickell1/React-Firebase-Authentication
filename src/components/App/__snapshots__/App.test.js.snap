// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AreaChart should render AreaChart component 1`] = `ShallowWrapper {}`;
